<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRiwayat</class>
 <widget class="QWidget" name="FormRiwayat">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>600</width> <height>500</height> </rect> </property>
  <property name="windowTitle"> <string>Data Riwayat</string> </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry"> <rect> <x>50</x> <y>20</y> <width>450</width> <height>210</height> </rect> </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0"><widget class="QLabel" name="l1"><property name="text"><string>ID Riwayat</string></property></widget></item>
    <item row="0" column="1"><widget class="QLineEdit" name="editID"/></item>
    <item row="1" column="0"><widget class="QLabel" name="l2"><property name="text"><string>ID User</string></property></widget></item>
    <item row="1" column="1"><widget class="QLineEdit" name="editIDUser"/></item>
    <item row="2" column="0"><widget class="QLabel" name="l3"><property name="text"><string>ID Barang</string></property></widget></item>
    <item row="2" column="1"><widget class="QLineEdit" name="editIDBrg"/></item>
    <item row="3" column="0"><widget class="QLabel" name="l4"><property name="text"><string>Jenis</string></property></widget></item>
    <item row="3" column="1"><widget class="QLineEdit" name="editJenis"/></item>
    <item row="4" column="0"><widget class="QLabel" name="l5"><property name="text"><string>Qty</string></property></widget></item>
    <item row="4" column="1"><widget class="QLineEdit" name="editQty"/></item>
    <item row="5" column="0"><widget class="QLabel" name="l6"><property name="text"><string>Keterangan</string></property></widget></item>
    <item row="5" column="1"><widget class="QLineEdit" name="editKet"/></item>
    <item row="6" column="0"><widget class="QLabel" name="l7"><property name="text"><string>Tanggal</string></property></widget></item>
    <item row="6" column="1"><widget class="QLineEdit" name="editTgl"/></item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan"> <property name="geometry"> <rect> <x>60</x> <y>250</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Simpan</string> </property> </widget>
  <widget class="QPushButton" name="btnHapus"> <property name="geometry"> <rect> <x>170</x> <y>250</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Hapus</string> </property> </widget>
  <widget class="QPushButton" name="btnUbah"> <property name="geometry"> <rect> <x>280</x> <y>250</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Ubah</string> </property> </widget>
  <widget class="QPushButton" name="btnBersih"> <property name="geometry"> <rect> <x>390</x> <y>250</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Bersih</string> </property> </widget>
  <widget class="QTableWidget" name="tabelRiwayat">
   <property name="geometry"> <rect> <x>50</x> <y>300</y> <width>500</width> <height>180</height> </rect> </property>
   <column><property name="text"><string>ID</string></property></column>
   <column><property name="text"><string>User</string></property></column>
   <column><property name="text"><string>Barang</string></property></column>
   <column><property name="text"><string>Jenis</string></property></column>
   <column><property name="text"><string>Qty</string></property></column>
  </widget>
 </widget>
 <resources/> <connections/>
</ui>
